src/index.ts

```diff
@@ -59,7 +59,7 @@ export const xplorersbot: HttpFunction = async (req, res) => {
                 .toLowerCase()
                 .startsWith("hey openai");
 
-            if (slackEvent?.type === "message") {
+            if (!isChannelOpenAI && slackEvent?.type === "message") {
```

Comment:
@geshan Sorry I had to this to avoid processing the event in openai channel and to avoid welcoming users in that channel

Reply:
All good